docs/source/models.rst

```diff
@@ -266,52 +266,58 @@ For more info, see:
     mlflow pyfunc serve --help
     mlflow pyfunc predict --help
 
-Deploy a ``python_function`` model on Microsoft AzureML
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Microsoft Azure ML
+^^^^^^^^^^^^^^^^^
+The :py:mod:`mlflow.azureml` module can package ``python_function`` models into Azure ML container images.
+These images can be deployed to Azure Kubernetes Service (AKS) and the Azure Container Instances (ACI)
+platform for real-time serving.
+
+* :py:func:`build_image <mlflow.azureml.build_image>` registers an MLflow model with an existing Azure ML
+  workspace and builds an Azure ML container image for deployment to AKS and ACI. The `Azure ML SDK`_ is
+  required in order to use this function.
```

Comment:
Mention that you need py3 here (kind of confusing that pip install just fails, and I was in a python2 conda env).